packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx

```diff
@@ -45,51 +61,267 @@ export class AtomicBreadcrumbManager {
+      </li>
+    ));
+    return moreButton
+      ? [...renderedBreadcrumbs, moreButton]
+      : renderedBreadcrumbs;
   }
 
   private get categoryFacetBreadcrumbs() {
-    return this.state.categoryFacetBreadcrumbs.map(
-      (breadcrumb: CategoryFacetBreadcrumb) => {
-        const pathString = breadcrumb.path
-          .map((value: CategoryFacetValue) => value.value)
-          .join('/');
-        return <button onClick={breadcrumb.deselect}>{pathString}</button>;
+    const breadcrumbs = this.state.categoryFacetBreadcrumbs.map(
+      (breadcrumb) => {
+        const breadcrumbsValues = this.getCategoryBreadrumbValues(breadcrumb);
+        const renderedBreadcrumbs = this.state.categoryFacetBreadcrumbs.every(
+          (value: CategoryFacetBreadcrumb) => value.path.length > 0
+        );
+        return renderedBreadcrumbs ? (
+          <ul part="breadcrumbs" class="breadcrumb p-0 m-0">
+            <li part="breadcrumb-label" class="text-muted">
+              {breadcrumb.field}:&nbsp;
+            </li>
+            {breadcrumbsValues}
+          </ul>
+        ) : (
+          ''
+        );
       }
     );
+    return breadcrumbs;
+  }
+
+  private getCategoryBreadrumbValues(values: CategoryFacetBreadcrumb) {
+    const breadcrumbsToShow = this.categoryCollapsedBreadcrumbsHandler(values);
+    const renderedBreadcrumbs = breadcrumbsToShow.map((breadcrumbValue) => (
+      <li
+        part="breadcrumb-value category-breadcrumb-value"
+        class="breadcrumb-item"
+      >
+        <button
+          part="breadcrumb-button"
+          class={this.buttonClasses}
+          onClick={values.deselect}
+        >
+          {breadcrumbValue}
+        </button>
+      </li>
+    ));
+    return [...renderedBreadcrumbs, this.mainClear];
   }
 
   render() {
+    if (this.hasActiveBreadcrumbs()) {
+      return (
+        <div class="row">
+          <span class=" col-9">
+            {this.facetBreadcrumbs}
+            {this.numericFacetBreadcrumbs}
+            {this.dateFacetBreadcrumbs}
+            {this.categoryFacetBreadcrumbs}
+          </span>
+          <span class="col-3 text-right">
+            {this.getClearAllFiltersButton()}
+          </span>
+        </div>
+      );
+    }
+  }
+
+  private getClearAllFiltersButton() {
     return (
-      <div class="breadcrumb-manager">
-        {this.facetBreadcrumbs}
-        {this.numericFacetBreadcrumbs}
-        {this.dateFacetBreadcrumbs}
-        {this.categoryFacetBreadcrumbs}
-      </div>
+      <button
+        part="breadcrumb-button"
+        class={this.buttonClasses}
+        onClick={() => {}} //TODO https://coveord.atlassian.net/browse/KIT-269
+      >
+        Clear All Filters
+      </button>
     );
   }
+
+  private showFacetCollapsedBreadcrumbs(field: string) {
+    this.collapsedBreadcrumbsState = {
+      ...this.collapsedBreadcrumbsState,
+      [field]: false,
+    };
+  }
+
+  private collapsedBreadcrumbsHandler<T extends BaseFacetValue>(
```

Comment:
I would breakdown this method in multiple simpler ones. It's also better to return early than handling multiple successive if/elses